cmake/Modules/LibAddPlugin.cmake

```diff
@@ -42,16 +42,18 @@ function (add_plugintest testname)
 
 
 		set (PLUGIN_NAME elektra-${testname})
-		restore_variable (${PLUGIN_NAME} ARG_LINK_LIBRARIES)
+		restore_variable (${PLUGIN_NAME} ARG_LINK_LIBRARIES ALLOW_MATCH)
 		restore_variable (${PLUGIN_NAME} ARG_COMPILE_DEFINITIONS)
 		restore_variable (${PLUGIN_NAME} ARG_INCLUDE_DIRECTORIES)
 		restore_variable (${PLUGIN_NAME} ARG_INCLUDE_SYSTEM_DIRECTORIES)
-		restore_variable (${PLUGIN_NAME} ARG_LINK_ELEKTRA)
+		restore_variable (${PLUGIN_NAME} ARG_LINK_ELEKTRA ALLOW_MATCH)
 		restore_variable (${PLUGIN_NAME} ARG_ADD_TEST)
 		restore_variable (${PLUGIN_NAME} ARG_INSTALL_TEST_DATA)
+		restore_variable (${PLUGIN_NAME} ARG_OBJECT_SOURCES)
```

Comment:
also mention here that it is for tests.

Reply:
This is used for the I/O adapters which are required for the plugin (not tests only).